spec/controllers/hyrax/embargoes_controller_spec.rb

```diff
@@ -26,6 +26,7 @@
       it 'redirects' do
         get :edit, params: { id: not_my_work }
         expect(response.status).to eq 302
```

Comment:
I think this line can be removed because `redirect_to` is already going to check the status.